src/Facility.AspNetCore/Facility.AspNetCore.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <OutputType>Library</OutputType>
```

Comment:
This is needed because of the `FrameworkReference`?

Reply:
It probably isn't necessary.